files/default/audit_report.rb

```diff
@@ -15,15 +15,16 @@ def report
         interval_enabled = node['audit']['interval']['enabled']
         interval_time = node['audit']['interval']['time']
         report_file = node['audit']['output']
+        test_array = node['audit']['profiles']
```

Comment:
Is `profiles` maybe a better name? I was thinking about an array of single tests first